lib/common/shared/src/main/scala/net/wiringbits/common/models/Email.scala

```diff
@@ -19,5 +21,7 @@ object Email extends WrappedString.Companion[Email] {
       }
   }
 
-  override def trusted(string: String): Email = new Email(string)
+  implicit override def trusted(string: String): Email = new Email(string)
+
+  implicit val sqlType: String = "CITEXT"
```

Comment:
This doesn't make much sense to me, can we write it somewhere else? the typo layer would be ideal